src/index.html

```diff
@@ -1,17 +1,180 @@
+                            <h5 class="gallery__project-type">Han lars ben</h5>
+                            <p class="gallery__description">Lucas ipsum dolor sit amet organa sidious mace jade kenobi moff mace skywalker windu yavin. Zabrak palpatine mon kenobi darth mothma skywalker dantooine. Wicket solo solo bespin hoth antilles.</p>
+                        </figcaption>
+                        <picture>
+                            <source srcset="./img/architecture3.webp" type="image/webp">
+                            <source srcset="./img/architecture3.jpg" type="image/jpeg">
+                            <img class="gallery__photo" srcset="./img/architecture3.jpg" alt="Library project">
+                        </picture>
+                    </figure>
+                </li> 
+
+            </ul>
+            <ul class="gallery__btn-container">
+                <button class="gallery__btn" id="gallery__btn1"><a href="#gallery__element1"></a></button>
+                <button class="gallery__btn" id="gallery__btn2"><a href="#gallery__element2"></a></button>
+                <button class="gallery__btn" id="gallery__btn3"><a href="#gallery__element3"></a></button>
+            </ul>
+    </section>
+
+    <!-- Contact form -->
+    <section class="contact" id="contact">
+        <h6 class="contact__heading">Napisz do mnie</h6>
+        <form class="contact__form">
+            <input class="contact__email" type="email" placeholder="wpisz swój adres email">
+            <textarea class="contact__msg" name="message" rows="4" cols="50" placeholder="wpisz treść wiadomości"></textarea>
+            <button class="contact__btn">Wyślij</button>
+        </form>
+    </section>
+
+    <!-- Footer -->
+    <footer>
+        <section class="info">
+            <div class="info__contact">
+                <a href="mailto:adres@email.pl" class="info__email">adres@email.pl</a>
+                <h6 class="info__phone-number">+48 000 111 222</h6>
+            </div>
+                <div class="info__go-up">
+                    <ion-icon name="chevron-up-outline" class="info__up-icon"></ion-icon>
+                </div>
+            <ul class="info__social-media">
+                <li><a href="https://www.linkedin.com/" target="_blank" rel="noopener noreferrer"><ion-icon class="info__social-media-icon" name="logo-linkedin"></ion-icon></a></li> 
+                <li><a href="https://www.facebook.com/" target="_blank" rel="noopener noreferrer"><ion-icon class="info__social-media-icon" name="logo-facebook"></ion-icon></a></li>
+                <li><a href="https://www.instagram.com/" target="_blank" rel="noopener noreferrer"><ion-icon class="info__social-media-icon" name="logo-instagram"></ion-icon></a></li>
+            </ul>
+        </section>
+    </footer>
+    <script rel="preconnect" type="module" src="https://unpkg.com/ionicons@5.4.0/dist/ionicons/ionicons.esm.js"></script>
```

Comment:
tutaj powinna być inna relacja :) preconnect oznacza, że to co wskazujesz powinno być poprane jako pierwsze np. fonty, czy cssy potrzebne do wygenerowania atf (czyli 1 co widzi użytkownik, ale to tutaj nie jest konieczne). W Twoim przypadku tylko fonty :)